climada/engine/unsequa/calc_base.py

```diff
@@ -292,8 +311,21 @@ def _make_uniform_base_sample(self, N, problem_sa, sampling_method,
         #c.f. https://stackoverflow.com/questions/2724260/why-does-pythons-import-require-fromlist
         import importlib # pylint: disable=import-outside-toplevel
         salib_sampling_method = importlib.import_module(f'SALib.sample.{sampling_method}')
-        sample_uniform = salib_sampling_method.sample(
-            problem = problem_sa, N = N, **sampling_kwargs)
+
+        if sampling_method == 'ff': #the ff sampling has a fixed sample size and
+                                    #does not require the N parameter
+            if (problem_sa['num_vars'] & (problem_sa['num_vars'] - 1) != 0):
+                raise ValueError("The number of parameters must be a power of 2 "
```

Comment:
```suggestion
                raise ValueError("The number of parameters must be a power of 2. "
```

Reply:
Do you suggest simply adding a . after the 2 or also removing everything that comes after it?

Reply:
Just add the '.' Otherwise the message will look incomplete.